contracts/bonding/BondingManager.sol

```diff
@@ -276,11 +280,11 @@ contract BondingManager is IBondingManager, Manager {
         // Current round must be initialized
         require(roundsManager().currentRoundInitialized());
 
-        if (transcoderStatus(msg.sender) == TranscoderStatus.Unbonding) {
+        if (transcoderStatus(msg.sender) == TranscoderStatus.Unbonded) {
```

Comment:
Good catch. Was this a bug before?

Reply:
Yeah...caught while running the node. Will need to write more thorough test suites